sdk/python/kfp/compiler/v2_compat.py

```diff
@@ -100,10 +100,16 @@ def update_op(op: dsl.ContainerOp,
       op.name,
       "--pipeline_name",
       pipeline_name,
-      "--pipeline_run_id",
-      "$(WORKFLOW_ID)",
-      "--pipeline_task_id",
+      "--run_id",
+      "$(KFP_RUN_ID)",
+      "--run_resource",
+      "workflows.argoproj.io/$(WORKFLOW_NAME)",
```

Comment:
Do we need this `workflows.argoproj.io/` here?

Reply:
It's meant to make the field vendor-neutral, it can also be a tekton pipeline with KFP-tekton